mcs_benchmark_data/models/benchmark.py

```diff
@@ -14,4 +18,23 @@ class Benchmark(_Model):
     abstract: str
     author: Tuple[str, ...]
     dataset: Tuple[BenchmarkDataset, ...]
-    submissions: Tuple[Submission, ...]
\ No newline at end of file
+    submissions: Tuple[Submission, ...]
+
+    def to_rdf(
+        self, *, graph: Graph, **kwds) -> Resource:

```

Comment:
You don't need `**kwds` since your signature only has `graph: Graph`. I just did that in paradicms because most implementations didn't care about `property_definitions`.

Reply:
Taking a look at property_definitions, I assumed that they were only needed for the types of objects in paradicms and aren't needed in this implementation at all. Was I correct?

Reply:
Yes, I use those to add arbitrary (and opaque) properties to objects. Not something we need here.